ledger/participant-integration-api/src/main/scala/platform/store/backend/h2/H2StorageBackend.scala

```diff
@@ -396,6 +402,8 @@ private[backend] object H2StorageBackend
       partyArrayContext = partyArrayContext,
       witnessesWhereClause =
         arrayIntersectionWhereClause("tree_event_witnesses", Set(requestingParty)),
+      submittersInPartiesClause = arrayIntersectionWhereClause("submitters", Set(requestingParty)),
+      columnEqualityBoolean = columnEqualityBoolean("event_kind", "20"),
```

Comment:
How about renaming this parameter to `createEventFilter = columnEqualityBoolean("event_kind", "20"),` the generic "columnEqualityBoolean" name makes it difficult to understand what value needs to be passed in.

Reply:
yeah it is a boolean derived in the select clause only (it is a good question why we need this boolean at all as we could just add a where clause to do the same in the queries that care about it, no?  here i just focused on getting functional equivalent to boolean used for postgres/h2 for oracle, since oracle does not have a native boolean type, so you have to do 0/1 numeric conversion sorcery

Reply:
renamed as suggested